doc-content/enterprise-only/employee-rostering/er-skills-proc.adoc

```diff
@@ -5,16 +5,16 @@ You can set all _skills_ that can be necessary in any position within the roster
 
 .Procedure
 
-. To enter or change skills in the Optaweb employee rostering application web interface, click the *Skills* tab.
+. To enter or change skills in the OptaWeb employee rostering application web interface, click the *Skills* tab.
 +
 You can see the numbers of currently visible skills in the top right part of the browser window, for example, `1-15 of 34`. You can use the *<* and *>* buttons to display other skills in the list.
 +
 You can enter any part of a skill name in the *Search* box to search for skills.
 +
 . Complete the following steps to add a new skill:
-.. Click *Add Skill*.
-.. Enter the name of the new skill in the text field under *Skill name*.
-.. Click *Save Skill*.
+.. Click *Add*.
+.. Enter the name of the new skill in the text field under *Name*.
+.. Click the Save icon.
```

Comment:
Question - does this use a floppy disk icon or is it titled?

Suggestion - bold name if titled or use lowercase if not:

s/Click the Save icon./Click the save icon/Click the *Save* icon.

Reply:
Same as "Employee", icon but no title.